src/util/shuffle.ts

```diff
@@ -0,0 +1,9 @@
+// function to randomly shuffle an array
+export function shuffle(array) {
```

Comment:
It would be better if this function creates a new array and returns it rather than shuffling the array in place.

Reply:
Done, please check @harshkhandeparkar